crates/router/src/core/user_role.rs

```diff
@@ -628,6 +629,7 @@ pub async fn list_users_in_lineage(
                     merchant_id: Some(&user_from_token.merchant_id),
                     profile_id: None,
                     version: None,
+                    limit: request.limit,
```

Comment:
For now we can omit limit in Request, let's pass it as `None`, since the limit in DB query need not to dependent on request always.
It's good that we tested with this.

Reply:
@apoorvdixit88: just here or in all 3 calls to `ListUserRolesByOrgIdPayload`?

Reply:
All three calls and don't accept it in request.

Reply:
Done in 5a1d2b3633e4fc8862f61bed9b5810eac2a313cb.